api/main.go

```diff
@@ -243,16 +248,39 @@ func (ar *Runner) Stop() error {
 	return nil
 }
 
-func (ar *Runner) getMemberPubKey(ref string) string { //nolint
+func (ar *Runner) getMemberPubKey(ref string) (string, error) { //nolint
 	ar.cacheLock.RLock()
 	key, ok := ar.keyCache[ref]
 	ar.cacheLock.RUnlock()
 	if ok {
-		return key
+		return key, nil
+	}
+	args, err := core.MarshalArgs()
```

Comment:
I think you can just create empty core.Args{} without calling MarshalArgs

Reply:
MarshalArgs returns []byte